view/frontend/web/js/view/payment/method-renderer/tns-hpf.js

```diff
@@ -42,23 +44,26 @@ define(
                 template: 'OnTap_MasterCard/payment/tns-hpf',
                 active: false,
                 adapterLoaded: false,
+                showCcForm: false,
                 buttonTitle: null,
                 buttonTitleEnabled: $t('Place Order'),
                 buttonTitleDisabled: $t('Please wait...'),
                 imports: {
                     onActiveChange: 'active'
                 },
                 creditCardExpYear: '',
-                creditCardExpMonth: ''
+                creditCardExpMonth: '',
```

Comment:
Extraneous comma?

Reply:
Thank you for mentioning it! I will fix the issue and update the Pull Request.